stacks/environment-aws/eks.tf

```diff
@@ -47,7 +48,24 @@ EOF
       protect_from_scale_in = true
       enabled_metrics       = ["GroupMinSize", "GroupMaxSize", "GroupDesiredCapacity", "GroupInServiceInstances", "GroupPendingInstances", "GroupStandbyInstances", "GroupTerminatingInstances", "GroupTotalInstances"]
       pre_userdata          = local.ssm_init
-    },
+      kubelet_extra_args    = var.enable_spot_instances ? "--node-labels=kubernetes.io/lifecycle=normal --register-with-taints=${var.ondemand_toleration_key}=true:NoSchedule" : ""
+    },   
+  ]
+
+  worker_groups_launch_template_mixed = [
```

Comment:
should there be 3, one for each AZ?

Also, can you add `      enabled_metrics       = ["GroupMinSize", "GroupMaxSize", "GroupDesiredCapacity", "GroupInServiceInstances", "GroupPendingInstances", "GroupStandbyInstances", "GroupTerminatingInstances", "GroupTotalInstances"]
`  like you see on the ondemand clusters?

Reply:
Added `enabled_metrics`.  Also added spots across 3 AZs, in case any workloads that get run on spots require a PV/PVC tied to a specific AZ.